src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -300,6 +300,20 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
             selectedAsset: assetMetadata,
             assets: _.values(this.props.project.assets),
         });
+
```

Comment:
I don't believe this code should be needed... the childAssets are pull out already in the `onAssetMetadataChanged` function and set on the state.

Reply:
the selectedAsset state must still be set here